gateway-ha/src/main/resources/logback.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE configuration>
+
+<configuration>
+    <import class="ch.qos.logback.core.ConsoleAppender" />
+    <import class="ch.qos.logback.core.filter.EvaluatorFilter" />
+
+    <appender name="STDOUT" class="ConsoleAppender">
+        <filter class="EvaluatorFilter">
+            <evaluator>
+                <expression>event.getMessage().contains("Authorization")</expression>

```

Comment:
What happens if we use broken Java expressions here? Does CI catch such mistake?

Reply:
I would assume no. It looks like it print the error message in the console, but CI won't fail